src/data/TargetDate.ts

```diff
@@ -0,0 +1 @@
+export const target = new Date("12/31/2024 23:59:59");
```

Comment:
Could move this into calender.ts and export it there, just so we don't make a whole new file for one line. That's my bad I didn't comment about this earlier.

Reply:
Done !